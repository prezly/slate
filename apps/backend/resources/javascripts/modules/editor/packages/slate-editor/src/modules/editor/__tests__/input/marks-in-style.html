<p style="font-style: italic;"><span style="vertical-align: sub;">italic subscript</span><b>bold italic</b><u><b>bold italic underlined</b></u><sup style="font-weight: 600;">bold italic superscript</sup></p>