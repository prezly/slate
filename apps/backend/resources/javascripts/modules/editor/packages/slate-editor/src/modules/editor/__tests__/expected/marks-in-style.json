[
    {
        "type": "paragraph",
        "children": [
            {
                "text": "italic subscript",
                "subscript": true,
                "italic": true
            },
            {
                "text": "bold italic",
                "bold": true,
                "italic": true
            },
            {
                "text": "bold italic underlined",
                "bold": true,
                "italic": true,
                "underlined": true
            },
            {
                "text": "bold italic superscript",
                "bold": true,
                "italic": true,
                "superscript": true
            }
        ]
    }
]
